static/docs/command-reference/import.md

```diff
@@ -1,9 +1,11 @@
 # import
 
-Download or copy file or directory from any <abbr>DVC project</abbr> in a Git
-repository (e.g. hosted on GitHub) into the <abbr>workspace</abbr>, and track
-changes in this [external dependency](/doc/user-guide/external-dependencies).
-Creates [DVC-files](/doc/user-guide/dvc-file-format).
+Import file or directory from any <abbr>DVC project</abbr> in a Git
```

Comment:
it's too long for the summary

Reply:
all these details should be part of the Description section

Reply:
Also I think we don't want to use the word import to define what import is 🙂

Reply:
Rewritten intro (and fixed file formatting).